nearcore/src/runtime/mod.rs

```diff
@@ -614,9 +614,12 @@ impl NightshadeRuntime {
             .unwrap()
             .install(|| {
                 contract_codes.par_iter().for_each(|code| {
+                    // FIXME: fetch the correct protocol version somehow?
+                    let current_protocol_version = ProtocolVersion::MAX;
```

Comment:
Note to self: this is wrong in the absolute terms, but maybe fine in practice? Need to think more about this and write propper comment. here.

Reply:
hmm why don't we simply use `PROTOCOL_VERSION`?

Reply:
:facepalm: because I thought that `ProtocolVersion` is a proper struct, and so `ProtocolVersion::MAX` (which I saw used elsewhere) is the correct way to refer to current protocol version, while `PROTOCOL_VERSION` is some temporary constant we use in a specific test. 

But of course that's the other way around -- ProtocolVersion is just u32, and we have `PROTOCOL_VERSION` top-level const because we can't say `ProtocolVersion::latest()`.

Reply:
:facepalm: :facepalm: :facepalm: 

There's `let protocol_version =` at the start of this function that gets the right protocol version.